### PR TITLE
flatten directories

### PR DESCRIPTION
**Pull request by mlautman**
Removing pr2_tutorials directory and placing its contents in the parent directory

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -53,7 +53,7 @@
 # Add any paths that contain custom themes here, relative to this directory.
 
 # Links
-extlinks = {'codedir': ('https://github.com/ros-planning/moveit_tutorials/tree/kinetic-devel/doc/pr2_tutorials/%s', ''),
+extlinks = {'codedir': ('https://github.com/ros-planning/moveit_tutorials/tree/kinetic-devel/doc/%s', ''),
```
Comment (mlautman) [inline conf.py]: I'm not sure how extlinks is being used but there are some subdirectories in doc that do not contain code and I'm not sure if they will be effected. catkin build and the build_locally.sh script still work.

Comment (davetcoleman) [reply conf.py]: i believe extlinks are shortcuts you can use in URLs throughout the tutorials

if some are no longer applicable, delete them

File: index.rst [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+These tutorials will run you through how to use MoveIt! with your robot. It is assumed that you have already configured MoveIt! for your robot - check the `list of robots running MoveIt! <http://moveit.ros.org/robots/>`_ to see whether MoveIt! is already available for your robot. Otherwise, skip to the tutorial on Setting up MoveIt! for your robot. If you just want to test MoveIt!, use the Panda as your quick-start robot.
 
-.. note:: All tutorials referencing the PR2 have only been tested with ROS Indigo but likely work with ROS Jade. See `issue <https://github.com/ros-planning/moveit/issues/110>`_ for more information.
+.. note:: All tutorials referencing the Panda have only been tested with ROS Kinetic but likely work with ROS Jade and Indigo. See `issue <https://github.com/ros-planning/moveit/issues/110>`_ for more information.
```
Comment (davetcoleman) [inline index.rst]: i think this should say something like The Panda tutorials are only for Kinetic and newer. For old versions of ROS see the PR2 tutorials here

File: conf.py (hunk 1 above)
Comment (mlautman) [reply conf.py]: Thanks! If I change ros-planing to PickNikRobotics and kinetic-devel to directory-refactor the links remain intact. I reverted the branch change in anticipation of merging

**Comment by mlautman** [comment]
ready for second pass

File: index.rst [hunk 3]
```diff
@@ -3,9 +3,7 @@ MoveIt! Tutorials
 
 These tutorials will run you through how to use MoveIt! with your robot. It is assumed that you have already configured MoveIt! for your robot - check the `list of robots running MoveIt! <http://moveit.ros.org/robots/>`_ to see whether MoveIt! is already available for your robot. Otherwise, skip to the tutorial on Setting up MoveIt! for your robot. If you just want to test MoveIt!, use the Panda as your quick-start robot.
 
-.. note:: All tutorials referencing the Panda have only been tested with ROS Kinetic but likely work with ROS Jade and Indigo. See `issue <https://github.com/ros-planning/moveit/issues/110>`_ for more information.
-
-Previous version of these tutorials: `ROS Indigo <http://docs.ros.org/indigo/api/moveit_tutorials/html/>`_
+.. note:: All tutorials referencing the Panda have only been tested with ROS Kinetic. For earlier versions of ROS (eg. indigo) See `the ROS Indig tutorials with the PR2 here <http://docs.ros.org/indigo/api/moveit_tutorials/html/doc/ikfast_tutorial.html>`_
```
Comment (davetcoleman) [inline index.rst]: Indig spell error

**Comment by mlautman** [comment]
ready for review

**Pull request merged**